target.go

```diff
@@ -35,8 +35,9 @@ import (
 
 type targetServer struct {
 	verbose     bool
-	resolver    *targetResolver
+	resolver    []*targetResolver
 	odohKeyPair odoh.ObliviousDNSKeyPair
+	telemetryClient *telemetry
 }
 
 func decodeDNSQuestion(encodedMessage []byte) (*dns.Msg, error) {
```

Comment:
Are we still using this function? If not, can we delete it?

Reply:
This is still used for DOH queries.